pkg/api/v1beta4/dynakube/convert_from.go

```diff
@@ -13,91 +13,91 @@ import (
 )
 
 // ConvertFrom converts from the Hub version (latest) to this version (v1beta4).
-func (dst *DynaKube) ConvertFrom(srcRaw conversion.Hub) error {
+func (dk *DynaKube) ConvertFrom(srcRaw conversion.Hub) error {
```

Comment:
I am not sure about this one. Should we `nolint` this? I mean yes of course is violates this check, but on the other hand I am not sure how confusing it can become when trying to distinguish between the `dynakube` versions.

cc @andriisoldatenko

Reply:
Yes, tbh I agree, would also go with nolinting it